app/helpers/writable_helper.rb

```diff
@@ -68,8 +68,8 @@ def simple_format_largebreak(text, options={})
   BR = '<br>'.freeze
 
   # specific blockquote handling is due to simple_format wanting to wrap a blockquote in a paragraph
-  def sanitize_written_content(content)
-    unless content[P_TAG] || content[BR_TAG]
+  def sanitize_written_content(content, editor_mode='html')
+    unless editor_mode == 'rtf' || content[P_TAG] || content[BR_TAG]
```

Comment:
My brain is hurting with this, but shouldn't it be

```suggestion
    unless editor_mode == 'html' || content[P_TAG] || content[BR_TAG]
```
? The situations where the content contains P_TAG or BR_TAG are exactly the situations when we should declare editor_mode = html, I believe

Reply:
Waaaait, no, HTML is the fake HTML that doesn't contain actual markup, I seeeeeeee

Okay this makes sense, I was just getting confused by the editor mode meaning